beacon_node/beacon_chain/src/attestation_simulator.rs

```diff
@@ -0,0 +1,93 @@
+use crate::{BeaconChain, BeaconChainTypes};
+use slog::{debug, error};
+use slot_clock::SlotClock;
+use std::sync::Arc;
+use task_executor::TaskExecutor;
+use tokio::time::sleep;
+use types::Slot;
+
+/// Spawns a routine which produces an unaggregated attestation at every slot.
+///
+/// This routine will run once per slot
+pub fn start_attestation_simulator_service<T: BeaconChainTypes>(
+    executor: TaskExecutor,
+    chain: Arc<BeaconChain<T>>,
+) {
+    executor.clone().spawn(
```

Comment:
Ask Paul how to properly 'gate' this. If I understood correctly from the discription, we should gate this when the validator_monitor is 'active'. Could you please define what you mean by active as the beacon chain cannot start with a validator monitor to my understanding.

Reply:
Oh yeah, good question. I think we could add a function to the validator monitor like:

```rust
fn is_enabled(&self) -> bool {
  self.auto_register || !self.validators.is_empty()
}
```

Reply:
Actually, let's just forget about the gating and run this all the time. It's very low overhead and I think it's interesting data to have on all nodes.